### PR TITLE
Send mails when delivered_date is update

### PR DESCRIPTION
**Pull request by diegotc86**
Don't forget to configure the ".env" file with your credentials.

I had difficulties using Insomnia so I created a temporal Edit View for admin/shipments in order to make a PATCH request to the server.

You need to be logged as admin in order to access: http://localhost:3000/admin/shipments/:id/edit

### CODE REVIEW

File: app/controllers/admin/shipments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  
+  class ShipmentsController < ApplicationController
+
+    def edit
+      @shipment = Shipment.find(params[:id])
+    end
+
+    def update
+      shipment = Shipment.find(params[:id])
+      sender = Sender.find(shipment.sender_id)
```
Comment (Linzeur) [inline app/controllers/admin/shipments_controller.rb]: It is not necessary to create a variable and call to Sender model to do that, you can use your variable shipment of the following way: "shipment.sender"

File: app/controllers/admin/shipments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      recipient = User.find(shipment.user_id)
+      if shipment.update(shipment_params)
+        ShipmentMailer.with(shipment: shipment, sender: sender, recipient: recipient).shipment_delivered.deliver_now
```
Comment (Linzeur) [inline app/controllers/admin/shipments_controller.rb]: In addition to my previous comment, I think you don't have to pass 3 new params, with only shipment to can access to information of the others.

File: app/controllers/admin/shipments_controller.rb (hunk 1 above)
Comment (diegotc86) [reply app/controllers/admin/shipments_controller.rb]: Thanks Brayan, I will implement it

File: app/controllers/admin/shipments_controller.rb (hunk 2 above)
Comment (diegotc86) [reply app/controllers/admin/shipments_controller.rb]: Thanks Brayan, I will implement it

**Pull request merged**